src/path_info.cpp

```diff
@@ -158,6 +158,12 @@ void PATH_INFO::set_standard_filenames(void)
     update_pathname("obsolete-mods", FILENAMES["moddir"] + "obsolete-mods.json");
     update_pathname("defaultsounddir", FILENAMES["datadir"] + "sound");
 
+    // User's data directories
+    update_pathname("user_datadir", FILENAMES["user_dir"] + "data/");
+    update_pathname("user_gfxdir", FILENAMES["user_datadir"] + "gfx/");
```

Comment:
> The default gfx dir lives under /gfx, not /gfx, see line 122 in this method.

Don't understand this feedback. They are separate directories, and the location of the gfxdir depends on DATA_DIR_PREFIX. 122 puts gfxdir under base_path. 118 puts gfxdir under datadir. This line puts **user_gfxdir** under user_datadir, mirroring the behavior at 118, except in a user modifiedable directory instead of a system dir.

> This will break your check in build_tilesets_list()

Which check?

> also cause the creation of a phantom directory in the default install.

That's the idea. It creates a directory that the user can easily locate to install mods. See BevapDin's comment.